input/pagecontent/reference.md

```diff
@@ -1544,25 +1651,103 @@ A defined rule set can be applied to an item by using an [`insert` rule](#rule-s
 
-Invariants are incorporated into a profile via `obeys` rules explained [above](#invariant-rules).
+Value sets are defined using the declarative keyword `ValueSet`, with optional keywords `Id`, `Title` and `Description`
 
-**Example:**
+Codes must be taken from one or more terminology systems (also called code systems or vocabularies). Codes cannot be defined inside a value set. If necessary, [you can define your own code system](#defining-code-systems).
 
-* Define an invariant found in US Core using FSH:
+The contents of a value set are defined by a set of rules. There are four types of rules to populate a value set:
+
+> **Note:** In value set rules, the word `include` is optional.
+
+| To include... | Syntax | Example |
+|-------|---------|----------|
+| A single code | `* include {Coding}` | `* include SCT#961000205106 "Wearing street clothes, no shoes"` <br/> or equivalently, <br/> `* SCT#961000205106 "Wearing street clothes, no shoes"`|
+| All codes from a value set | `* include codes from valueset {ValueSet name|id|url}` | `* include codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason`  <br/> or equivalently, <br/> `* codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason`|
+| All codes from a code system | `* include codes from system {CodeSystem name|id|url}` | `* include codes from system http://snomed.info/sct` <br/> or equivalently, <br/> `* codes from system http://snomed.info/sct`|
+| Selected codes from a code system (filters are code system dependent) | `* include codes from system {CodeSystem name|id|url} where {filter} and {filter} and ...` | `* include codes from system SCT where concept is-a #254837009` <br/> or equivalently, <br/> `* codes from system SCT where concept is-a #254837009`|
+{: .grid }
+
+See [below](#filters) for discussion of filters.
+
+Analogous rules can be used to leave out certain codes, with the word `exclude` replacing the word `include`:
+
+| To exclude... | Syntax | Example |
+|-------|---------|----------|
+| A single code | `* exclude {Coding}` | `* exclude SCT#961000205106 "Wearing street clothes, no shoes"` |
+| All codes from a value set | `* exclude codes from valueset {ValueSet name|id|url}` | `* exclude codes from valueset http://hl7.org/fhir/ValueSet/data-absent-reason` |
+| All codes from a code system | `* exclude codes from system {CodeSystem name|id|url}` | `* exclude codes from system http://snomed.info/sct` |
+| Selected codes from a code system (filters are code system dependent) | `* exclude codes from system {CodeSystem name|id|url} where {filter}` | `* exclude codes from system SCT where concept is-a #254837009` |
+{: .grid }
+
+##### Filters
+
+A filter is a logical statement in the form `{property} {operator} {value}`, where operator is chosen from [a value set](http://hl7.org/fhir/ValueSet/filter-operator) containing the values:
```

Comment:
Suggest calling out the VS by name rather than just "a value set":
> A filter is a logical statement in the form `{property} {operator} {value}`, where operator is chosen from **the [FilterOperator value set](http://hl7.org/fhir/ValueSet/filter-operator)** containing the values:

Reply:
Being specific is good.  I removed the the list of codes since that could change and the link is sufficient.